app/controllers/movies_controller.rb

```diff
@@ -7,7 +7,12 @@ def index
   end
 
   # GET /movies/1 or /movies/1.json
-  def show; end
+  def show
+    @seances = Seance
+               .where(movie_id: params[:id])
+               .where(start_time: DateTime.current + 30.minutes..)
+               .order(:start_time)
```

Comment:
I would personally specify if you order by start_time ascending or descending, even if this behavies as you want it to by default.
Dates can get really confusing so it might pay to be more descriptive with what we are doing with them
But it's not a major issue
```suggestion
               .order(:start_time, :desc)
```

Reply:
Makes sense, I'm going to specify things like that from now on :).